bazel/repository_locations.bzl

```diff
@@ -105,12 +105,12 @@ REPOSITORY_LOCATIONS_SPEC = dict(
         project_name = "Abseil",
         project_desc = "Open source collection of C++ libraries drawn from the most fundamental pieces of Google’s internal codebase",
         project_url = "https://abseil.io/",
-        version = "17c954d90d5661e27db8fc5f086085690a8372d9",
-        sha256 = "2e4ace2ed32a4ccfd29e856ad72b4fd1eae2ec060d3ba8646857fa170d6e8269",
+        version = "20211102.0",
```

Comment:
This change includes serious regressions against msvc, which is required for consideration of adoption. You can review the azure build log for details. Not only should it be -std:c99 on windows vs -std=c99, but there are significant clib deviations that absl made contrary assumptions about. In our experience MS has a slightly different impl where the spec was left ambiguous. Does not lgtm, deps change cannot be accepted.

Reply:
@wrowe any ideas on ways forward on this, grepping etc im not seeing where this flag is set, but i guess there are further issues anyway

im assuming that we do want to update, at least at some point, maybe that is incorrect

looking at the error log, it seems to be something to do with quic, not sure if that is worth looking at